docs/resources/etc_hosts_allow.md.erb

```diff
@@ -0,0 +1,63 @@
+---
+title: About the etc_hosts_allow Resource
+---
+
+# etc_hosts_allow
+
+Use the `etc_hosts_allow` InSpec audit resource to test rules set to accept daemon and client traffic set in /etc/hosts.allow file.
+## Syntax
+
+An etc/hosts.allow rule specifies one or more daemons mapped to one or more clients,
+with zero or more options to use to accept traffic when found.
+
+## Syntax
+
+Use the where clause to match a property to one or more rules in the hosts.allow file.
+
+    describe etc_hosts_allow.where { property == ['values'] } do
+      its ('client_list') { should eq [['values']] }
+      its ('options') { should eq [['values']] }
+    end
+
+Use the optional constructor parameter to give an alternative path to hosts.allow
+
+    describe etc_hosts_allow(hosts_path).where { property == ['values'] } do
+      its ('client_list') { should eq [['values']] }
+      its ('options') { should eq [['values']] }
+    end
+
+where
+
+* `daemon_list` is a list of daemons that will be allowed to pass traffic in.
+* `client_list` is a list of clients will be allowed to pass traffic in.
+* `options` is a list of tasks that to be done with the rule when traffic is found.
+
+## Supported Properties
+
+    'daemon_list', 'client_list', 'options'
+
```

Comment:
It is under my impression that cmp only works with arrays of one value. So for this resource I think we should leave the matcher operator as eq because in most cases rules specify more than one daemon, client or option.